README.md

```diff
@@ -214,7 +214,7 @@ There are also some built in 'views' that will create a special column.  For exa
 
     { view: "increment" , label: false , text: "+" }
 
-will have a link that increments the quantity. Setting the `label:false` will hide the label for the view. You can specify the text of the link with that `text:` attribute.
+will have a link that increase the quantity. Setting the `label:false` will hide the label for the view. You can specify the text of the link with that `text:` attribute.
```

Comment:
Firstly, this is not a correction of spelling OR grammar. You have actually MADE a grammar error in your poor attempt at correction, and shown a lack of understanding.

"Increments" is correct. While you could substitute "increases" for some loss of domain specific meaning, you cannot substitute "increase" which is grammatically INCORRECT.

Trivial PRs like this can be annoying, but when they are wrong, they are more than annoying. If you want to correct someone's English, make sure you know enough to do so!